### PR TITLE
Tooltip changes

### PR DESCRIPTION
**Pull request by emre2038**
## Description
modifying the tooltip depending on the choropleth method

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-493

## How is it tested automatically?
added `__tests__/gui/popup.feature`

## How should a reviewer test it locally
* select an indicator which is `subindicator` choropleth method and confirm that the tooltip and the mapchip are modified accordingly
* select an indicator which is `sibling` choropleth method and confirm that the tooltip and the mapchip are modified accordingly
* select an indicator which is `absolute` choropleth method and confirm that the tooltip and the mapchip are modified accordingly

## Screenshots
* subindicator choropleth method
image

* sibling choropleth method
image

* absolute choropleth method
image

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

File: __tests__/gui/popup/popup.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+});
+
+When('I move the cursor', () => {
+    cy.get('body')
+        .trigger('mousemove', {clientX: 500, clientY: 358})
+        .trigger('mousemove', {clientX: 120, clientY: 130});
+})
+
+When('I hover over the center of the map', () => {
+    hoverOverTheMapCenter();
```
Comment (michaelglenister) [inline __tests__/gui/popup/popup.js:43]: I really like how these cypress tests follow DRY principles

File: __tests__/gui/popup.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    And I hover over the center of the map
+    Then I confirm that the tooltip is created correctly if the choropleth method is sibling
+
+    # choropleth method absolute
+    And I move the cursor
+    And I click on "South African Citizenship" in Data Mapper
+    And I click on "Citizenship" in Data Mapper
+    And I click on "Yes" in Data Mapper
+    And I hover over the center of the map
+    Then I confirm that the tooltip is created correctly if the choropleth method is absolute
```
Comment (michaelglenister) [inline __tests__/gui/popup.feature:32]: Can this be a little more descriptive, than "created correctly" for example 

Then I confirm that the tooltip is created with the correct text and the label is hidden

File: src/index.html [hunk 3]
```diff
@@ -22,7 +22,8 @@
   <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
   <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
   <link href="images/webclip.png" rel="apple-touch-icon">
-  <script defer="" src="https://kit.fontawesome.com/95e41092af.js" crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/95e41092af.js" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="custom-css/mapchip.scss"><link rel="stylesheet" href="custom-css/popup.scss"></head>
```
[the remark this answers is not in the document]
Comment (emre2038) [reply src/index.html]: done 👍

File: __tests__/gui/popup.feature (hunk 2 above)
Comment (emre2038) [reply __tests__/gui/popup.feature]: done 👍

File: src/index.html [hunk 4]
```diff
@@ -22,7 +22,10 @@
   <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
   <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
   <link href="images/webclip.png" rel="apple-touch-icon">
-  <script defer="" src="https://kit.fontawesome.com/95e41092af.js" crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/95e41092af.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="custom-css/mapchip.scss">
+  <link rel="stylesheet" href="custom-css/popup.scss">
+</head>
```
Comment (michaelglenister) [inline src/index.html:28]: This \</head> can be removed. Yamyam

**Review by michaelglenister — COMMENTED** [review]
One small fix and then it's approved
